### PR TITLE
Merging new code. 

### PR DESCRIPTION
**Pull request by theAndrewCline**
@harikakurma please update this field with a description of what you coded. Thanks!

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (theAndrewCline) [inline .vscode/launch.json]: you should review this comment.

File: src/components/Completed.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+class completedTask extends Component {
+  state = {};
```
Comment (theAndrewCline) [inline src/components/Completed.jsx]: removed unused state, you don't need it.

File: src/components/MyHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        };
+      });
+    }
+  };
+
+  nullWarning() {
+    let addWarning = "btn m-2 btn-";
+    addWarning += this.state.input === "" ? "warning" : "success";
+    return addWarning;
+  }
```
Comment (theAndrewCline) [inline src/components/MyHeader.jsx]: it's a good idea to put line between your functions. it's much cleaner.

File: src/components/MyHeader.jsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    console.log(selectedTask);
+    this.setState({
+      tasks: filterTask,
+      input: selectedTask.content,
+      id: id,
+      editItem: true
+    });
+  };
+
+  // // emptyInput() {
```
Comment (theAndrewCline) [inline src/components/MyHeader.jsx]: you should remove commented code before committing.

File: src/components/heading.jsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  constructor() {
+    super();
+    this.state = { add: "" };
+    // this.event = this.event.bind(this);
+  }
+  addText = todo => {
+    this.setState({ add: todo.target.value });
+  };
+  showText = () => {
+    //document.getElementById("task").innerText = this.state.add;
```
Comment (theAndrewCline) [inline src/components/heading.jsx]: you should remove commented out code.

File: src/components/heading.jsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+class heading extends Component {
+  constructor() {
+    super();
+    this.state = { add: "" };
+    // this.event = this.event.bind(this);
```
Comment (theAndrewCline) [inline src/components/heading.jsx]: you should remove commented out code.

Comment (harikakurma) [reply src/components/heading.jsx]: @theAndrewCline ok can I do that after completing this app, I just want to make sure where I came from for practice purpose.

File: src/components/MyHeader.jsx (hunk 3 above)
Comment (harikakurma) [reply src/components/MyHeader.jsx]: @theAndrewCline sure will follow that.

**Pull request closed without merging**